sycl/test/abi/sycl_symbols_windows.dump

```diff
@@ -685,15 +685,18 @@
 ?AccessTargetMask@handler@_V1@sycl@@0HB
 ?Clear@exception_list@_V1@sycl@@AEAAXXZ
 ?DirSep@OSUtil@detail@_V1@sycl@@2QEBDEB
+?DisableRangeRounding@detail@_V1@sycl@@YA_NXZ
 ?DisableRangeRounding@handler@_V1@sycl@@AEAA_NXZ
 ?EventMode@SubmissionInfo@v1@detail@_V1@sycl@@QEAAAEAW4event_mode_enum@experimental@oneapi@ext@45@XZ
 ?EventMode@SubmissionInfo@v1@detail@_V1@sycl@@QEBAAEBW4event_mode_enum@experimental@oneapi@ext@45@XZ
 ?GDBMethodsAnchor@SampledImageAccessorBaseHost@detail@_V1@sycl@@IEAAXXZ
 ?GDBMethodsAnchor@UnsampledImageAccessorBaseHost@detail@_V1@sycl@@IEAAXXZ
+?GetRangeRoundingSettings@detail@_V1@sycl@@YAXAEA_K00@Z
 ?GetRangeRoundingSettings@handler@_V1@sycl@@AEAAXAEA_K00@Z
 ?HasAssociatedAccessor@handler@_V1@sycl@@AEBA_NPEAVAccessorImplHost@detail@23@W4target@access@23@@Z
 ?PushBack@exception_list@_V1@sycl@@AEAAX$$QEAVexception_ptr@std@@@Z
 ?PushBack@exception_list@_V1@sycl@@AEAAXAEBVexception_ptr@std@@@Z
+?RangeRoundingTrace@detail@_V1@sycl@@YA_NXZ

```

Comment:
Why do we need to export both: `detail::RangeRoundingTrace()` and `handler::RangeRoundingTrace()`?